src/api/schema/events/log.rs

```diff
@@ -0,0 +1,37 @@
+use crate::event;
+use async_graphql::Object;
+use chrono::{DateTime, Utc};
+use serde_json::json;
+
+#[derive(Debug)]
+/// A log event
```

Comment:
```suggestion
/// A log event.
```

Reply:
This is another 'magic' GraphQL doc comment.